Mathlib/MeasureTheory/Measure/Haar/OfBasis.lean

```diff
@@ -226,6 +226,24 @@ instance IsAddHaarMeasure_basis_addHaar (b : Basis ι ℝ E) : IsAddHaarMeasure
   rw [Basis.addHaar]; exact Measure.isAddHaarMeasure_addHaarMeasure _
 #align is_add_haar_measure_basis_add_haar IsAddHaarMeasure_basis_addHaar
 
+instance [TopologicalSpace.SecondCountableTopology E] (b : Basis ι ℝ E) :
+    SigmaFinite b.addHaar := by
+  rw [Basis.addHaar_def]; exact MeasureTheory.Measure.sigmaFinite_addHaarMeasure
+
+/-- Let `b` and `b'` two bases of `E`. The basis `b'` defines the same Haar measure as the basis
+`b` iff the parallelepiped defined by `b'` has measure `1` for the Haar measure defined by `b`. -/
+theorem Basis.addHaar_eq_iff {ι' : Type*} [Fintype ι'] [TopologicalSpace.SecondCountableTopology E]
+    {b : Basis ι ℝ E} {b' : Basis ι' ℝ E} :
+    b.addHaar = b'.addHaar ↔ b.addHaar b'.parallelepiped = 1 := by
+  rw [Basis.addHaar_def, Basis.addHaar_def]
+  exact Measure.addHaarMeasure_eq_iff b.parallelepiped b'.parallelepiped
+
+@[simp]
+theorem Basis.addHaar_reindex {ι ι' E : Type*} [Fintype ι] [Fintype ι'] [NormedAddCommGroup E]
```

Comment:
```suggestion
theorem Basis.addHaar_reindex {E : Type*} [NormedAddCommGroup E]
```
Is `E` needed, or can yo ure-use the global one?

Reply:
Yes, I did a loose job of tracking what was or was not already declared. It should be better now.